### PR TITLE
Add preact example (using cli)

### LINKED ISSUE #270: Add preact-cli example
This is the now.json needed:

```js
{
    "version": 2,
    "name": "tcs-aux",
    "builds": [
        {"src": "package.json", "use": "@now/static-build", "config": { "distDir": "build" }}
    ],
    "routes": [
        { "src": "^/assets/(.*)", "dest": "/assets/$1" },
        { "src": "^/(.*)\\.(.*)\\.(js|css|json)$", "headers": {"cache-control": "Immutable"}, "dest": "/$1.$2.$3" },
        { "src": "^/(.*)\\.(js|css|json)$", "dest": "/$1.$2" },
        { "src": "^/(.*)", "dest": "/index.html" }
    ]
}
```

### PR DESCRIPTION
**Pull request by sync**
fixes #270

Deployed here: https://preact-h2jc0typx.now.sh/

### CODE REVIEW

File: preact-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  ],
+  "routes": [
+    { "src": "^/assets/(.*)", "dest": "/assets/$1" },
+    {
+      "src": "^/(.*)\\.(.*)\\.(js|css|json|css.map|js.map)$",
+      "headers": { "cache-control": "Immutable" },
+      "dest": "/$1.$2.$3"
+    },
+    { "src": "^/(.*)\\.(js|css|json|css.map|js.map)$", "dest": "/$1.$2" },
+    { "src": "^/(.*)", "dest": "/index.html" }
```
Comment (sync) [inline preact-cli/README.md]: needed to add `css.map` and `js.map`

**Comment by timothyis** [comment]
Thank you so much for your contribution! Do you have any opinions on whether the directory name should be `preact-cli` or just `preact`? `preact-cli` is the tool used to generate the project, but the project is, in the end, Preact.

**Comment by sync** [comment]
happy to rename it preact

**Comment by timothyis** [comment]
I think that would be the way to do it

**Comment by sync** [comment]
doing this now

**Comment by sync** [comment]
done, waiting for someone to auth deploy, thanks

**Comment by timothyis** [comment]
Hi @sync. Sorry for the delay here. I'll be testing this tomorrow to see if we can merge it in. 

One topic before hand though: What do you think about extending this example to be a realistic project. For example; following the static example (https://github.com/zeit/now-examples/tree/master/static), where we pull in 5 Hacker News posts from their API and displaying them.

The example could even go further by having a post page showing comments, using the power of Preact.

Is this something you'd be interested in working on?

**Comment by sync** [comment]
yes leave it to me, wanted to make sure there was interest for the simple before doing this. Also been working on preact x support i think it would end up being nicer overall. I'll work on it this week end.

**Comment by timothyis** [comment]
Perfect, thank you @sync! 😍

**Comment by sync** [comment]
@timothyis been working on a reason react example but once i am done with it i'll back port it to preact and js. It's a hacker news client with a graphql server. Both running in the same repository using yarn workspaces.

**Comment by msweeneydev** [comment]
Hey @sync, thank you for your contribution here. As you might have seen, the examples have been recently updated, including a Preact example, with no configuration required.

As a result, I am closing this PR. Thank you again for your efforts, we'd love to get your thoughts on the new examples!

**Pull request closed without merging**